task3-4/task3-4/EvaluateScreen/EvaluateViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  EvaluateViewController.swift
+//  task3-4
+//
+//  Created by Григоренко Александр Игоревич on 03.10.2022.
+//
+
+import UIKit
+
+/// Экран оценки приложения.
+final class EvaluateViewController: UIViewController {
+    
+    // MARK: Visual components
+    
+    var evaluateScreenLabel = UILabel()
```

Comment:
```suggestion
  private  var evaluateScreenLabel = UILabel()
```
private - вышел из чата?)

Reply:
XD